pkg/arvo/lib/verb.hoon

```diff
@@ -62,39 +62,40 @@
 ++  on-peek
   |=  =path
   ^-  (unit (unit cage))
-  %-  (print bowl "{<dap.bowl>}: on-peek on path {<path>}")
+  %-  (print bowl |.("{<dap.bowl>}: on-peek on path {<path>}"))
   (on-peek:ag path)
 ::
 ++  on-agent
   |=  [=wire =sign:agent:gall]
   ^-  (quip card:agent:gall agent:gall)
-  %-  (print bowl "{<dap.bowl>}: on-agent on wire {<wire>}, {<-.sign>}")
+  %-  (print bowl |.("{<dap.bowl>}: on-agent on wire {<wire>}, {<-.sign>}"))
   =^  cards  agent  (on-agent:ag wire sign)
   [[(emit-event %on-agent wire -.sign) cards] this]
 ::
 ++  on-arvo
   |=  [=wire =sign-arvo]
   ^-  (quip card:agent:gall agent:gall)
-  %-  (print bowl "{<dap.bowl>}: on-arvo on wire {<wire>}, {<[- +<]:sign-arvo>}")
+  %-  %+  print  bowl  |.
+      "{<dap.bowl>}: on-arvo on wire {<wire>}, {<[- +<]:sign-arvo>}"
   =^  cards  agent  (on-arvo:ag wire sign-arvo)
   [[(emit-event %on-arvo wire [- +<]:sign-arvo) cards] this]
 ::
 ++  on-fail
   |=  [=term =tang]
   ^-  (quip card:agent:gall agent:gall)
-  %-  (print bowl "{<dap.bowl>}: on-fail with term {<term>}")
+  %-  (print bowl |.("{<dap.bowl>}: on-fail with term {<term>}"))
   =^  cards  agent  (on-fail:ag term tang)
   [[(emit-event %on-fail term) cards] this]
 --
 ::
 ++  print
-  |=  [=bowl:gall =tape]
+  |=  [=bowl:gall tape=_^?(|.(*tape))]
   ^+  same
   =?  .  bowl-print
     %-  (slog >bowl< ~)
     .
   ?.  loud  same
-  %-  (slog leaf+tape ~)
+  %-  (slog leaf+(tape) ~)
```

Comment:
While I'm nitpicking, I think `$:tape` would be idiomatic here. (Also, the face `tape` has become questionable.)

Reply:
`=mold` delenda est? 🤔